### DISCUSSION TITLE
Hide areas/rooms for some persons in DD

### DISCUSSION THREAD

**Discussion by kris-tant** [dwainscheeren/dwains-lovelace-dashboard#785] [General]
I want to maintain only 1 dashboard, and limit what some users can see this dashboard. For example, in DD, can I hide the view of some areas to some people? With normal dashboards, this is possible with state-switch cards.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
